services/addons/images/rsu_status_check/crontab

```diff
@@ -2,4 +2,6 @@ PYTHONPATH=/home
 * * * * * /usr/local/bin/python3 /home/rsu_ping_fetch.py
 * * * * * /usr/local/bin/python3 /home/rsu_pinger.py
 0 0 * * * /usr/local/bin/python3 /home/purger.py
-0 */4 * * * /usr/local/bin/python3 /home/rsu_snmp_fetch.py
+0 */4 * * * /usr/local/bin/python3 /home/rsu_msgfwd_fetch.py
+0 */1 * * * /usr/local/bin/python3 /home/rsu_security_fetch.py
+*/20 * * * * /usr/local/bin/python3 /home/rsu_health_fetch.py
```

Comment:
These cron values were chosen based on how frequently it would make sense to update these various statuses. SCMS certificate granularity is only down to 1 hour so updating it every hour makes sense. Health statuses could change quicker than every 20 minutes but we have to keep in mind that CDOT has 600 RSUs we need to run and wait for potential 5 second timeouts on SNMP commands. This is why these were written to run in parallel, but I am being precautious here.